### PR TITLE
Add EIP-5794: eth/69: add withdrawals to block

### PR DESCRIPTION
**Pull request by MariusVanDerWijden**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by MariusVanDerWijden** [comment]
We might not do this after all, since we didn't create a new version when we added the basefee to the header

File: EIPS/eip-5794.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+With the introduction of withdrawals, the header and the block body format is changed. In order to allow nodes to respond with a valid block, the block bodies need to be extended with the withdrawal list and the block header with the withdrawal root.
+
+The `block-body` definition is used in the `BlockBodies` message. The `block` definition is used by the `NewBlock` message. The `header` definition is used in the `block` definition and the `BlockHeaders` message.
+
+## Specification
+
+Modify the following message type from the `eth` protocol:
+
+* `block-body`:
+* * **(eth/67)**: We defined `block-body = [transactions, ommers]`
```
Comment (xinbenlv) [inline EIPS/eip-5794.md:28]: Could you help establish `ommers` in this doc? By either proper reference or explanation?

File: EIPS/eip-5794.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+This EIP changes the `eth` protocol and requires rolling out a new version, `eth/69`. Supporting multiple versions of a wire protocol is possible. Rolling out a new version does not break older clients immediately, since they can keep using protocol version `eth/68`.
+
+This EIP does not change consensus rules of the EVM and does not require a hard fork.
+
+## Security Considerations
+
+None
+
+## Copyright
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
```
Comment (xinbenlv) [inline EIPS/eip-5794.md:88]: This is an old template, remember do update.

File: EIPS/eip-5794.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+The `block-body` definition is used in the `BlockBodies` message. The `block` definition is used by the `NewBlock` message. The `header` definition is used in the `block` definition and the `BlockHeaders` message.
+
+## Specification
+
+Modify the following message type from the `eth` protocol:
+
+* `block-body`:
+* * **(eth/67)**: We defined `block-body = [transactions, ommers]`
+* * **(eth/68)**: We define `block-body = [transactions, ommers, withdrawals]`
```
Comment (axic) [inline EIPS/eip-5794.md:29]: Aren't these `eth/69`?

File: EIPS/eip-5794.md (hunk 1 above)
Comment (axic) [inline EIPS/eip-5794.md:28]: Shouldn't this just refer to the previous proto?

```suggestion
* * **(eth/68)**: We defined `block-body = [transactions, ommers]`
```

File: EIPS/eip-5794.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+eip: 5794
+title: eth/69 - Add withdrawals to block
+description: Adds the withdrawals to the block bodies and the withdrawal root to the block header in the eth protocol
+author: Marius van der Wijden (@MariusVanDerWijden)
+discussions-to: tbd
+status: Draft
+type: Standards Track
+category: Networking
+created: tbd
+requires: 2464, 2481, 4938
```
Comment (Gabriel-Trintinalia) [inline EIPS/eip-5794.md:11]: Does it require EIP-5793 (eth/68) as well?

**Comment by MariusVanDerWijden** [comment]
Am going to close this, we decided that adding withdrawals to the block does not warrant a new protocol version.

**Pull request closed without merging**